### ISSUE TITLE
Scalar Dashboard: Auto-"Fit Domain to Data" Buttom

### ISSUE THREAD

**Issue by Wilann** [tensorflow/tensorboard#4500]
During training, I constantly have to click the "Fit Domain to Data" button to see changes in the graph (as the line often goes out of view). I'd be nice to see it automatically done. Is there any plan to implement such a feature?

**Comment by rmothukuru** [comment]
@Wilann,
In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thanks!

**Comment by Wilann** [comment]
There is no specific code - any scalar graph has the button "Fit Domain to Data". I'm just wondering if there are any plans to make it automatic.

**Comment by NachoLiu** [comment]
I have the same problem, when the data has been updated(when I train the model and add more data to tensorboard), and I refresh tensorboard, the graph didn't auto fit the data, I need to click the button "Fit Domain to Data",

**Comment by nfelt** [comment]
Thanks for the question - I think this issue may be a duplicate of #1946?  If that's the case please see the most recent update here: https://github.com/tensorflow/tensorboard/issues/1946#issuecomment-747822683

tl;dr: for the new Time Series dashboard, which includes the Scalars dashboard functionality as a subset, the chart domains should indeed update automatically on reload as you're requesting.  Could you give that a try and let us know how it works for you?

**Comment by psybuzz** [comment]
Closing this as a duplicate of https://github.com/tensorflow/tensorboard/issues/1946.  Please let us know if this is not the case!

**Comment by Wilann** [comment]
I think my issue was a duplicate of #1946. 

I can't figure out how to use the new Time Series dashboard with PyTorch - can't even find the documentation page.

**Comment by nfelt** [comment]
The Time Series dashboard doesn't have documentation yet (it's still quite new, though cc @psybuzz maybe we should have a tracking issue to add docs of some kind?).  But it doesn't require any special setup to use - if you have data visible in the scalars, image, or histogram dashboards they should show up under the Time Series tab automatically.

**Comment by psybuzz** [comment]
> maybe we should have a tracking issue to add docs of some kind?

Definitely a good idea.  Just filed https://github.com/tensorflow/tensorboard/issues/4660

As nfelt mentioned, the dashboard should appear in the header automatically
whenever scalar data, or histograms or images are present. In the meantime
before our docs are ready, please feel free to ask questions.

**Comment by Wilann** [comment]
I'm on Ubuntu 20.04 and TensorBoard v2.4.1 and it doesn't show up in the header for me. 

![Screenshot from 2021-02-08 19-18-00](https://user-images.githubusercontent.com/40744757/107298044-a4c9b380-6a42-11eb-9a78-841d4ec79b45.png)
![Screenshot from 2021-02-08 19-20-43](https://user-images.githubusercontent.com/40744757/107298117-c0cd5500-6a42-11eb-81c0-9148c5cd5eb4.png)

**Comment by psybuzz** [comment]
Ahh, thanks for the screenshot.  The Time Series dashboard is available within the local TensorBoard, activated by `tensorboard --logdir /some/path` and opening `localhost:6006`.

On the other hand, TensorBoard.dev the publicly hosted service, does not yet have Time Series available.

**Comment by Wilann** [comment]
Thank you for the clarification! Looks great and works just as expected!
